utility/serial.cpp

```diff
@@ -92,6 +92,71 @@ void ArdusatSerial::begin(unsigned long baud, bool setXbeeSpeed)
   }
 }
 
+/**
+ * Begin serial communications with a Sparkfun BlueSMiRF module at the specified baud
+ * rate.
+ *
+ * Note that baud rates above ~57600 are not well-supported by SoftwareSerial, and 
+ * even 57600 may cause some bugs.
+ *
+ * @param baud rate for serial communications
+ *
+ * NOTE: If this function is called multiple times with different values for 'baud'
+ *       without losing power in between calls, the baud rate will not be updated
+ *       after the first call.
+ */
+void ArdusatSerial::beginBluetooth(unsigned long baud)
+{
+  bool validBaud = true;
+  char baudCmd[8];
+
+  // Temporarily sets baud rate until power loss
+  switch (baud) {
+    case 1200:  strcpy(baudCmd, "U,1200,N"); break;
+    case 2400:  strcpy(baudCmd, "U,2400,N"); break;
+    case 4800:  strcpy(baudCmd, "U,4800,N"); break;
+    case 9600:  strcpy(baudCmd, "U,9600,N"); break;
+    case 19200: strcpy(baudCmd, "U,192K,N"); break;
+    case 38400: strcpy(baudCmd, "U,384K,N"); break;
+    case 57600: strcpy(baudCmd, "U,576K,N"); break;
```

Comment:
If we're already going through the trouble of copying into a buffer, let's put all these commands into prog mem so they don't eat away at our RAM footprint.

This looks like:

```c
const char baud_1200_cmd[] PROGMEM = "U,1200,N";
char baud_cmd[9];
strcpy_P(baud_cmd, baud_1200_cmd);
```